tools/size_report

```diff
@@ -0,0 +1,803 @@
+#!/usr/bin/env python3
+#
+# Copyright (c) 2016, 2020 Intel Corporation
+#
```

Comment:
I don't think that we can accept this file into the repository with the Intel copyright.  If so, it will need to be declared as a third party file.

Reply:
We already have declare Intel copyright in LICENSE file, so it's just simple to list this new script there.